python/pynq/intf/intf_const.py

```diff
@@ -37,6 +37,8 @@
 
 # Microblaze constants
 BIN_LOCATION = os.path.dirname(os.path.realpath(__file__)) + "/"
+ARDUINO = 3
+INTERFACE_ID = [ARDUINO]
```

Comment:
I will remove these constants. Here I assume users know the ID, i.e. constants like ARDUINO=3 will be specified by users.